macosx/BlocklistDownloader.mm

```diff
@@ -15,7 +15,7 @@ @interface BlocklistDownloader ()
 @property(nonatomic) blocklistDownloadState fState;
 
 - (void)startDownload;
-- (void)decompressFrom:(NSURL*)file to:(NSURL*)destination error:(NSError**)error;
+- (BOOL)decompressFrom:(NSURL*)file to:(NSURL*)destination error:(NSError**)error;
```

Comment:
Also seems to be unrelated to the title of the PR

Reply:
This fixes a warning:
>Method accepting NSError** should have a non-void return value to indicate whether or not an error occurred

I think it's fine in this PR. Let me know if you prefer that I move it to a separate PR, @DevilDimon.